src/components/app.js

```diff
@@ -0,0 +1,11 @@
+import Display from './display';
+import ButtonPanel from './button_panel';
+
```

Comment:
- [ ] OPTIONAL - Consider renaming this filename from app.js to App.js. It's a convention to capitalize and carmelize components filename.